app/src/main/java/com/onewelcome/showcaseapp/feature/userauthentication/implicitauthentication/ImplicitAuthenticationScreen.kt

```diff
@@ -0,0 +1,259 @@
+package com.onewelcome.showcaseapp.feature.userauthentication.implicitauthentication
+
+
+
+
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.Button
+import androidx.compose.material3.Checkbox
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.RadioButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavController
+import com.github.michaelbull.result.Result
+import com.github.michaelbull.result.onFailure
+import com.github.michaelbull.result.onSuccess
+import com.onegini.mobile.sdk.android.model.entity.UserProfile
+import com.onewelcome.core.components.SdkFeatureScreen
+import com.onewelcome.core.components.ShowcaseCard
+import com.onewelcome.core.components.ShowcaseFeatureDescription
+import com.onewelcome.core.components.ShowcaseStatusCard
+import com.onewelcome.core.components.ShowcaseTooltip
+import com.onewelcome.core.theme.Dimensions
+import com.onewelcome.core.theme.toErrorResultString
+import com.onewelcome.core.util.Constants
+import com.onewelcome.showcaseapp.R
+
+@Composable
+fun ImplicitAuthenticationScreen(
+    navController: NavController,
+    viewModel: ImplicitAuthenticationViewModel = hiltViewModel()
+) {
+    ImplicitAuthenticationScreenContent(
+        uiState = viewModel.uiState,
+        onNavigateBack = { navController.popBackStack() },
+        onEvent = { viewModel.onEvent(it) }
+    )
+}
+
+@Composable
+private fun ImplicitAuthenticationScreenContent(
+    onNavigateBack: () -> Unit,
+    uiState: ImplicitAuthenticationViewModel.State,
+    onEvent: (ImplicitAuthenticationViewModel.UiEvent) -> Unit,
+) {
+    //ListenForPinNavigationEvent(navigationEvents, onNavigateToPinScreen)
+    LoadData(onEvent)
```

Comment:
There's no need to load data from the screen. You can use init {} block in viewmodel. Please take a look how it is done for example in `SdkResetViewModel`